msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md

```diff
@@ -8,77 +8,137 @@ ms.localizationpriority: high
+Apps must not ask users to make payments within the Teams interface (see the following image). App shouldn't transmit financial instrument details to users through a bot interface.
+  ![In App Payments](~/assets/images/submission/validation-financial-information-1.png)
 
-* [Publisher Verification](/azure/active-directory/develop/publisher-verification-overview): Helps admins and end users understand the authenticity of app developers integrating with the Microsoft identity platform. When completed, a blue "verified" badge displays on the Azure Active Directory (Azure AD) consent dialog and other screens. For more information, see [frequently asked questions](/azure/active-directory/develop/publisher-verification-overview#frequently-asked-questions), [how to mark your app as publisher verified](/azure/active-directory/develop/mark-app-as-publisher-verified), and [troubleshoot publisher verification](/azure/active-directory/develop/troubleshoot-publisher-verification).
-* [Publisher Attestation](/microsoft-365-app-certification/docs/attestation): A process in which you share general, data handling, and security and compliance information to help potential customers make informed decisions about using your app.
 
-> [!NOTE]
-> If you're submitting an app that hasn't been listed previously, you can't officially complete Publisher Attestation until your app is in the Teams store. If you're updating a listed app, complete Publisher Attestation before you submit the latest version of the app.
+You may provide link to secure external payment services only if you made the appropriate disclosure in your terms of use, privacy policy, profile page, or website before the user agreed to use the app.
 
-### Bots
+No payment shall be made through an app for goods or services prohibited by General policy 100.10 Inappropriate content.
 
-For apps that use the Microsoft Azure Bot Service (such as bots and messaging extensions), you must follow all requirements defined in the Microsoft [Online Services Terms](https://www.microsoftvolumelicensing.com/DocumentSearch.aspx?Mode=3&DocumentTypeId=46).
+Apps running on the iOS or Android version of Teams must adhere to the following guidelines:
 
-Bots must always ask permission to upload a file and display a confirmation message after the file uploads.
+* Apps must not include in-app purchases, trial offers, or UI that aims to upsell users to paid versions or online stores to purchase or acquire other content, apps, or add-ins.  
+    ![In-app Purchases](~/assets/images/submission/validation-financial-information-in-app-purchase.png)
+    
+    :::image type="icon" source="~/assets/images/submission/validation-incorrect-icon.png"::: Mobile version of the app allows in-app purchase, has UI that aims to upsell to paid versions.
```

Comment:
Please change:

"Mobile version of the app allows in-app purchase, which has UI that aims to upsell to paid versions."